assets/js/event.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
+//var SpeakerPopover = document.getElementById("{{ speaker.slug }}");
+var currentSpeaker;
```

Comment:
Don’t use `var` - pick [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) or [`const`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const). Regardless, this variable should *not* be in the global scope. Use `x.extend()`